cirq-core/cirq/circuits/optimization_pass.py

```diff
@@ -92,11 +91,10 @@ def __init__(
             [Sequence['cirq.Operation']], ops.OP_TREE
         ] = lambda op_list: op_list,
     ) -> None:
-        """
-        Args:
-            post_clean_up: This function is called on each set of optimized
-                operations before they are put into the circuit to replace the
-                old operations.
+        """Args:
```

Comment:
First line should be a description of the method. In this case I think the args should be moved to `Attributes` on the class. https://google.github.io/styleguide/pyguide.html#384-classes